pcr/gizzmos.py

```diff
@@ -46,4 +46,78 @@ def adjust_volume(self, volume: float) -> str:
         return None, None
 
     def check_connection(self) -> bool:
-        return True
\ No newline at end of file
+        return True
+    
+
+import cv2 as cv
```

Comment:
Add these imports to the top of the file since that's where all of them are. Just good python coding procedure